Changes

```diff
@@ -63,6 +63,10 @@ Working version
 - #9316: Use typing information from Clambda for mutable Cmm variables.
   (Stephen Dolan, review by Vincent Laviron, Guillaume Bury and Xavier Leroy)
 
+- #9344, #9368: Prevent program built with "-output-complete-exe" from trying to
+  open themselves as bytecode file and crashing.
```

Comment:
I would insert something like `Disable exception backtraces in bytecode programs built with "-output-complete-exe"` here which may be more informative for the casual user.

Reply:
Indeed, I'll update the message